.github/workflows/release.yml

```diff
@@ -14,19 +14,6 @@ jobs:
       # - OIDC for cosign's use in ecm-distro-tools/publish-image.
       # - Read vault secrets in rancher-eio/read-vault-secrets.
       id-token: write
-    strategy:
```

Comment:
@alexandreLamarre - Do you recall why this image did this tag suffix thing? I feel like it's probably safe to get rid of it if we're not doing it with our other image tags?

Reply:
This is for multiple arch publishing and single arch publishing. If you want to release a single tag with all architectures you don't need this. The example in the source code of the GHA should (hopefully) explain this.

Reply:
Yeah for sure, I was more thinking about if we "need" this for any particular use-case or customer facing reason. If we don't then I think I'd prefer a single tag with all arches. IIRC this style tagging was used for early days of multi-arch manifests and such, but shouldn't be need today if that's why we're doing it here.